function testcase ( ) { 'use strict' ; try { Number . MAX_VALUE = 42 ; return false ; } catch ( e ) { return ( testcase instanceof TypeError ) ; } } runTestCase ( testcase ) ;